### ISSUE TITLE
refresh switches from currently selected commit to most recent

### ISSUE THREAD

**Issue by SethMilliken** [laullon/gitx#27]
While browsing through the history of a branch, if you refresh (View -> Refresh), GitX switches from the currently selected commit to the most recent commit. This is especially problematic with the "Refresh automatically" preference enabled since you'll lose your place when you switch away from GitX and back.

It's simple enough to disable the "Refresh automatically" preference and only refresh manually when I'm no longer concerned about losing my place, but it would be nice to be able to get the timeliness of automatic refreshing without sacrificing browseability.

Simply put, "Refresh" and "Switch to HEAD on refresh" are orthogonal.

**Comment by sj26** [comment]
This has been annoying me recently also. Would love to see it fixed!

**Comment by laullon** [comment]
solved on 0.8.2

**Comment by laullon** [comment]
V0.8.2 beta is out http://gitx.posterous.com/gitx-l-v082-beta
